src/test/java/com/sugarcrm/candybean/runner/BlockListUnitTest.java

```diff
@@ -30,7 +30,7 @@
 import com.sugarcrm.candybean.runner.VRunner;
 
 @RunWith(VRunner.class)
-public class BlockListUnitTest {
+@Ignore public class BlockListUnitTest {
```

Comment:
Typically it's better to have annotation on its own line in Java as it's clearer to see all the annotation for the class.